clock/system.go

```diff
@@ -60,6 +60,7 @@ func (st *systemTime) NewTicker(d time.Duration) Ticker {
 }
 
 func (st *systemTime) Tick(d time.Duration) <-chan time.Time {
+	//nolint: staticcheck // FIXME: SA1015: using time.Tick leaks the underlying ticker, consider using it only in endless functions, tests and the main package, and use time.NewTicker here
```

Comment:
> and use time.NewTicker here

We already have `func (st *systemTime) NewTicker(d time.Duration) Ticker`.
Does it mean, that we shouldn't use `Tick()` at all and deprecate it?

Reply:
Done.